Automattic/wp-calypso
# User Licensing: Add post-checkout UI for auto-activation flow

#### Changes proposed in this Pull Request

- This PR adds the post-checkout thank you page for users that have at least one jetpack site connected.
- Users select which site they want to activate their product on from a drop-down list of all their sites.
- Or they can choose, "I don't see my site. Let me configure it manually", in which they will be redirected to the manual activation flow.
- If the user does not have any connected Jetpack sites, they will skip this UI and be redirected straight to the "manual"-activation flow.

Related to 1201096622142517-as-1201096622142518

#### Testing instructions

- Create a JN site & connect to your account.
- Checkout this PR and `yarn-start`.
- Go to: http://localhost:3000/checkout/jetpack/jetpack_scan
- Complete purchase.  Verify you are redirected to: `/checkout/jetpack/thank-you/licensing-auto-activate/:product` and you can see the the new post-checkout UI (See screenshot below).
- In the dropdown select box, select, "I don't see my site. Let me configure it manually" and hit Continue.
- Verify you are taken to the manual license-activation UI flow (This UI will need more work in future PR's)
- Click the browsers "Back" button to navigate back to the auto-activation flow.
- This time in the dropdown select box, select your JN site that you connected earlier, and hit continue.
- Verify the Jetpack Scan subscription was successfully transferred to your JN site and you are redirected to the "<product> has been activated and is ready to go!". page.

#### Screenshots:

![Screenshot on 2021-11-01 at 16-48-55](https://user-images.githubusercontent.com/11078128/139740961-7a04cf7a-b2cc-4d74-a33d-c5367fca331a.png)

![Markup 2021-11-02 at 12 25 18](https://user-images.githubusercontent.com/11078128/139905577-0129c42c-4db0-47fe-b794-5597ff012312.png)
